routing/src/fib/fibobjects.rs

```diff
@@ -3,6 +3,8 @@
 
 //! Module that contains definitions and methods for fib objects
 
+use net::vxlan::Vni;
```

Comment:
I am really not a fan of the behavior added by this commit, seems really hacky and confusing for users to see prefixes that match but are not used in a VRF.  My understanding is we don't really use this and may remove it later.  Please correct me if I am wrong.

Reply:
Yes, we no longer need this as it is with the changes later in the branch.